projects/mvc/result-json/src/Program.cs

```diff
@@ -0,0 +1,74 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Hosting;
+using System;
+
+namespace PracticalAspNetCore
+{
+	public class Startup
+	{
+		public void ConfigureServices(IServiceCollection services)
+		{
+			services.AddControllersWithViews();
+
+		}
+
+		public void Configure(IApplicationBuilder app)
+		{
+			app.UseRouting();
+			app.UseEndpoints(endpoints => 
+			{
+				endpoints.MapDefaultControllerRoute();
+			});
+		}
+	}
+
+	public class HomeController : Controller 
+	{
+		IWebHostEnvironment _env;
+
+		public HomeController(IWebHostEnvironment env)
+		{
+			_env = env;
+		}
```

Comment:
Please create an index page so people can just run it and see a page. In this index page, put a one paragraph description and link it to the endpoint that return json.

Reply:
Added the index page in commit 829ff